modules/gltf/doc_classes/AssetDocument3D.xml

```diff
@@ -0,0 +1,108 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<class name="AssetDocument3D" inherits="Resource" is_experimental="true" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../../../doc/class.xsd">
+	<brief_description>
+	</brief_description>
+	<description>
+	</description>
+	<tutorials>
+	</tutorials>
+	<methods>
+		<method name="append_from_buffer">
+			<return type="int" enum="Error" />
+			<param index="0" name="bytes" type="PackedByteArray" />
+			<param index="1" name="base_path" type="String" />
+			<param index="2" name="state" type="GLTFState" />
```

Comment:
Move the GLTFState specific class to GLTFDocument and make a new append_from_buffer, etc [...]